.github/workflows/dusk-tests.yml

```
@@ -0,0 +1,183 @@
+name: Browser Tests (Dusk)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)